### PR TITLE
design is ready

### PR DESCRIPTION
**Pull request by magwas**

### CODE REVIEW

File: tidyup.zenta [hunk 1]
```diff
@@ -1979,799 +2088,1545 @@ They are brownish in color, to make it </documentation>
...
+          <bounds x="619" y="638" width="359"/>
+        </child>
+        <child xsi:type="zenta:DiagramObject" id="fd76a6f5-1a2c-4142-89d7-430689cf77ea" targetConnections="57b3c149-35b9-48e7-a8da-432da9b2b817" zentaElement="9332dd47-17c8-4ed8-86d6-a4b3295af46b">
+          <bounds x="621" y="584" width="359"/>
         </child>
       </element>
+    </folder>
+    <folder name="0. Architecture Relations" id="ae39d5b2-9ecf-496b-a0ed-d775187ca6f0">
+      <element xsi:type="zenta:BasicRelationship" id="90222234-25ea-4f26-9d32-3e452e0d5eb5" ancestor="1a16ba6a-0441-4e7d-8f9c-f6394fcdd10b" source="444fc5ce-7b74-48b2-b62d-ab160239d1cb" target="beceee57-a7b1-44c0-b535-d7e0efc4b62d"/>
+      <element xsi:
```
Comment (szirbucz) [inline tidyup.zenta]: description field is missing from CreateWorkRequestDTO

Comment (szirbucz) [inline tidyup.zenta]: Data types should have containing package

Comment (szirbucz) [inline tidyup.zenta]: WorkRequestListDTO was a restricted version of the WorkRequestDTO containing information needed for the listing. WorkRequestDTO should contain a description.

Comment (szirbucz) [inline tidyup.zenta]: What is this ADDRESS_ENTITY in the view of WorkRequestEntity type?

Comment (szirbucz) [inline tidyup.zenta]: Test Data is not human readable.

Comment (szirbucz) [inline tidyup.zenta]: USER_ID should be connected to ownerId. Isn't it?

Comment (magwas) [reply tidyup.zenta]: Because it is compiled from the other elements.

Comment (magwas) [reply tidyup.zenta]: done

Comment (magwas) [reply tidyup.zenta]: WorkRequestDTO is compiled, no need for description.
I designed WorkRequestListDTO to contain all returned information.
Maybe it is not a good idea for a real application. I prefer to leave it as is for now to be able to go fast.
Let's discuss what is the right solution for a real application.

Comment (magwas) [reply tidyup.zenta]: typo, deleted

Comment (magwas) [reply tidyup.zenta]: Refactored to four TestData classes. As test data is maybe the thing on which we have the tightest coupling, it is still quite crowded, but hopefully readable.

Comment (magwas) [reply tidyup.zenta]: that is OWNER_ID, this is where the connection was lost. fixed.

**Pull request merged**